clink/lua/src/settings_api.cpp

```diff
@@ -143,7 +143,7 @@ template <typename S, typename... V> void add_impl(lua_State* state, V... value)
 /// -name:  settings.add
 /// -ver:   1.0.0
 /// -arg:   name:string
-/// -arg:   default:...
+/// -arg:   default...: boolean | integer | string | table
```

Comment:
I prefer the original text for the following reasons:
- It should be `default` rather than `default...`.  The latter is a convention that means multiple values can be passed, but only one value can be passed.
- `...` for the type steers the reader to the rest of the text, which describes the variations and their respective requirements.  I would prefer not to go into detail describing the types there, while also leaving out details such as "table of strings".
- Listing types there can make it look as though color types are not supported, but they are.
- There may be new types added in the future, and the list is already long.